.github/workflows/demo_test.yml

```diff
@@ -32,15 +32,6 @@ jobs:
             SGX_MODE=SIM occlum build;
             occlum run /bin/hello_world"
 
-    - name: C with encrypted image test
```

Comment:
Since we have deleted this case, how to test if the fs image with a key can work?  Is there another test case that can substitute this one?

Reply:
The new added runtime_boot demo can cover the runtime mount encrypted rootfs image.
The remote_attestation/init_ra_flow can cover the general building encrypted image and acquiring key in init process. (In the HW CI)

Reply:
I think the runtime_boot demo should be added into the demo test CI

Reply:
Ok